conda_forge_tick/migrators.xsh

```diff
@@ -6,13 +6,15 @@ from conda.models.version import VersionOrder
 
 from rever.tools import (eval_version, indir, hash_url, replace_in_file)
 
-from .utils import render_meta_yaml
+from .utils import render_meta_yaml, convert_dict_to_nt
 
 
 class Migrator:
     """Base class for Migrators"""
     rerender = False
 
+    _class_version = 0
```

Comment:
I would prefer if this was called `migrator_version`, and it seems like this should be part of the public API